json-rpc/src/methods.rs

```diff
@@ -125,32 +123,40 @@ async fn get_account(
 ) -> Result<Option<AccountView>> {
     let address: String = serde_json::from_value(request.get_param(0))?;
     let account_address = AccountAddress::from_str(&address)?;
-    let response = service
+    let account_state_blob = service
         .db
         .get_account_state_with_proof_by_version(account_address, request.version())?
         .0;
+    if account_state_blob.is_none() {
+        return Ok(None);
+    }
+
+    let account_state = AccountState::try_from(&account_state_blob.unwrap())?;
```

Comment:
.unwrap() would make a lot of people nervous  (it panics) , I suggest we rewrite into an if-else or. an match Some()=> , None=>  that doesn't need unwrap()

Reply:
you can also write this way

let accoutn_state = account_state_blob.map(|x|  AccountState::try_from(x))? ;